data/harvesting.txt

```diff
@@ -8,6 +8,28 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+minable "meretidebris"
```

Comment:
These names should be "mereti debris" or "sestor debris" with a space, not smashed together.

Reply:
Thanks, I added a "_" between the names